.github/.templatesyncignore

```diff
@@ -0,0 +1,8 @@
+:!package.json
```

Comment:
ik denk dat het handig is om de .templatesyncignore file al toe te voegen aan de main branch. Dan kunnen de meest recente Brp.Shared.Infrastructure en Serilog.Sinks.PersistentFile worden gemerged in deze branch en in deze branch worden getest of alles correct werkt